.github/workflows/ci.yml

```
@@ -135,3 +143,12 @@ jobs:
         repo-type: MONOREPO
         mode: tag
         diff_dest: ${{ steps.calculate_changed_services.outputs.diff_dest }}
+
+  deploy-tmp-pr-docker-buildx-images:
+    if: ${{ github.event_name == 'pull_request' }}
+    needs: ci
+    uses: ./.github/workflows/run-plz.yaml
```

actionlint (1.7.4)

could not read reusable workflow file for "./.github/workflows/run-plz.yaml": open /home/jailuser/git/.github/workflows/run-plz.yaml: no such file or directory

(workflow-call)